tsconfig.json

```diff
@@ -5,6 +5,7 @@
   "compilerOptions": {
     "outDir": "dist-types",
     "incremental": false,
-    "sourceMap": true
+    "sourceMap": true,
+    "jsx": "react"
```

Comment:
do we want to keep this? i can add a trailing comma if so but i had to add this to get warnings in webstorm to go away